README.md

```diff
@@ -25,7 +25,8 @@ hesitate to read the source code and understand the mechanism of this bot.
 ## Exchange marketplaces supported
 
 - [X] [Bittrex](https://bittrex.com/)
-- [X] [Binance](https://www.binance.com/) ([*Note for binance users](#a-note-on-binance))
+- [X] [Binance](https://www.binance.com/) ([*Note for binance users](docs/exchanges.md#blacklists))
+- [X] [Kraken](https://kraken.com/)
```

Comment:
add a link to https://www.freqtrade.io/en/latest/exchanges/#kraken here

Reply:
i wouldn't (and did not do that intentionally) ... that's the overview page - the binance link is just there because it's important to exclude BNB ...

Reply:
okay